build/tools/mingw/parameters.bat

```diff
@@ -0,0 +1,50 @@
+@echo off
+
+rem Number of jobs for parallels builds
+set JOBS=3
```

Comment:
If the number of jobs is recommended to be set to "number of cores of your processor - 1", I wonder if the system environment variable `NUMBER_OF_PROCESSORS` could not be somehow used here to preset the number of jobs to some reasonable number?